src/reduced_basis/rb_scm_evaluation.C

```diff
@@ -529,7 +529,7 @@ void RBSCMEvaluation::legacy_read_offline_data_from_files(const std::string & di
 
   // Resize C_J based on C_J_stability_vector and Q_a
   C_J.resize( C_J_length );
-  for (std::size_t i=0; i<C_J.size(); i++)
+  for (auto i : index_range(C_J))
```

Comment:
This could also just be a plain range-for, right?

Reply:
Yep, I will change that one.